examples/animatediff/tests/test_save_video.py

```diff
@@ -0,0 +1,24 @@
+import glob
+import os
+import sys
+
+import numpy as np
+from PIL import Image
+
+__dir__ = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.abspath(os.path.join(__dir__, "..")))
+from animate_diff.utils.util import save_videos
+
+img_dir = os.path.join(__dir__, "../../videocomposer/datasets/webvid5")
```

Comment:
possible to move test data to a standardalone folder?

Reply:
it's to avoid enlarging the repo resize. for test script, reusing data from neighbor should be fine.